pywbem/cim_obj.py

```diff
@@ -2571,7 +2571,8 @@ def from_instance(class_, instance, namespace=None, host=None,
                 object. Thus, for example, if the class were retrieved from a
                 server, generally, the `LocalOnly` parameter in the request
                 should be `False` to assure that all superclass properties are
-                retrieved.
+                retrieved and the IncludeQualifiers should be set to `True`
```

Comment:
Use backquotes around "IncludeQualifiers", and insert "parameter" after it, consistent with the text for "LocalOnly".

Reply:
Done